node-fs-writefile/cp.js

```diff
@@ -0,0 +1,9 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+const argv = process.argv.slice(2);
+try {
+  const content = await readFile(argv[0], 'utf8');
+  await writeFile(argv[1], content.toString());
```

Comment:
If you use 'utf8' as an encoding in `readFile`, it will return a string, so there's no need to use `.toString()` here.

Note also, the code you have written works for text files, but not for binary files. Since you provide the 'utf8' encoding when reading the file, you will only be able to successfully read text files. To make this work for binary files (like images), you all have to do is remove the 'utf8' encoding!